customHttp.yml

```diff
@@ -0,0 +1,11 @@
+customHeaders:
+  - pattern: '**'
+    headers:
+      - key: 'Access-Control-Allow-Origin'
+        value: '*'
+      - key: 'Cache-Control'
+        value: 'public, max-age=1296000'
```

Comment:
is this global?
can this create a risk for any specific of new type of url in future getting cleared for a longer duration?
i think we can keep it for some specific paths only or file types

Reply:
Found the issue and have updated the cache policy now.